src/debug/HyprCtl.cpp

```diff
@@ -76,7 +76,7 @@ std::string monitorsRequest(eHyprCtlOutputFormat format, std::string request) {
     "vrr": {},
     "activelyTearing": {}
 }},)#",
-                m->ID, escapeJSONStrings(m->szName), escapeJSONStrings(m->szDescription), (m->output->make ? m->output->make : ""), (m->output->model ? m->output->model : ""),
```

Comment:
wait, don't change this! It's used by people to match monitors by `desc:` in rules :(

Reply:
That's why I changed it. The parenthesized DRM node name at the end is redundant (in both pretty and JSON output). The wiki says to ignore parenthesized name at the end, so I removed it. People probably wouldn't guess that internally, you just check if the `szDescription` starts with `szShortDescription` when matching `desc:szShortDescription`. They can still use the full description from `wlroots`, but now we can remove comments about it in the wiki. I think this should be more clear.

Reply:
oke